tst/parallel/echelon_form.tst

```diff
@@ -0,0 +1,15 @@
+gap> Read("read.g");
+gap> Read("main_seq_trafo.g");
+gap> Read("echelon_form.g");
+gap> dimension := 200;; rank := 150;; q := 5;; numberChops := 8;;
+gap> rs := RandomSource(IsMersenneTwister);;
+gap> echelon := echelonMat(dimension, dimension, rank, rs);;
+gap> shapeless := shapelessMat(echelon, dimension, dimension, rs, GF(q));;
+gap> result := ChiefParallel(GF(q), shapeless, numberChops, numberChops);;
+gap> result_std := EchelonMatTransformation(shapeless);;
+gap> -1 * result.vectors = result_std.vectors;
+true
+gap> -1 * result.coeffs = result_std.coeffs;
+true
+gap> -1 * result.vectors = MatrixByBlockMatrix(echelon);
+true
```

Comment:
The tests fail here!

Reply:
Okay, I jiust remembered that the rank is a separating invariant of equivalent matrices. So should we rather look at similar matrices? Can we compare the echelon forms then? Or do we not need to compare them anyway?